inclusion/blob_share_commitment_rules_test.go

```diff
@@ -326,86 +337,150 @@ func TestBlobMinSquareSize(t *testing.T) {
 	}
 }
 
+type subtreeWidthTestCase struct {
+	shareCount int
+	want       int
+}
+
+var subtreeWidthTestCases = []subtreeWidthTestCase{
+	{
+		shareCount: 0,
+		want:       1,
+	},
+	{
+		shareCount: 1,
+		want:       1,
+	},
+	{
+		shareCount: 2,
+		want:       1,
+	},
+	{
+		shareCount: defaultSubtreeRootThreshold,
+		want:       1,
+	},
+	{
+		shareCount: defaultSubtreeRootThreshold + 1,
+		want:       2,
+	},
+	{
+		shareCount: defaultSubtreeRootThreshold - 1,
+		want:       1,
+	},
+	{
+		shareCount: defaultSubtreeRootThreshold * 2,
+		want:       2,
+	},
+	{
+		shareCount: (defaultSubtreeRootThreshold * 2) + 1,
+		want:       4,
+	},
+	{
+		shareCount: (defaultSubtreeRootThreshold * 3) - 1,
+		want:       4,
+	},
+	{
+		shareCount: (defaultSubtreeRootThreshold * 4),
+		want:       4,
+	},
+	{
+		shareCount: (defaultSubtreeRootThreshold * 5),
+		want:       8,
+	},
+	{
+		shareCount: (defaultSubtreeRootThreshold * defaultMaxSquareSize) - 1,
+		want:       128,
+	},
+}
+
 func TestSubTreeWidth(t *testing.T) {
-	type testCase struct {
-		shareCount int
-		want       int
-	}
-	testCases := []testCase{
-		{
-			shareCount: 0,
-			want:       1,
-		},
-		{
-			shareCount: 1,
-			want:       1,
-		},
-		{
-			shareCount: 2,
-			want:       1,
-		},
-		{
-			shareCount: defaultSubtreeRootThreshold,
-			want:       1,
-		},
-		{
-			shareCount: defaultSubtreeRootThreshold + 1,
-			want:       2,
-		},
-		{
-			shareCount: defaultSubtreeRootThreshold - 1,
-			want:       1,
-		},
-		{
-			shareCount: defaultSubtreeRootThreshold * 2,
-			want:       2,
-		},
-		{
-			shareCount: (defaultSubtreeRootThreshold * 2) + 1,
-			want:       4,
-		},
-		{
-			shareCount: (defaultSubtreeRootThreshold * 3) - 1,
-			want:       4,
-		},
-		{
-			shareCount: (defaultSubtreeRootThreshold * 4),
-			want:       4,
-		},
-		{
-			shareCount: (defaultSubtreeRootThreshold * 5),
-			want:       8,
-		},
-		{
-			shareCount: (defaultSubtreeRootThreshold * defaultMaxSquareSize) - 1,
-			want:       128,
-		},
-	}
-	for i, tc := range testCases {
+	for i, tc := range subtreeWidthTestCases {
 		t.Run(fmt.Sprintf("shareCount %d", tc.shareCount), func(t *testing.T) {
 			got := inclusion.SubTreeWidth(tc.shareCount, defaultSubtreeRootThreshold)
 			assert.Equal(t, tc.want, got, i)
 		})
 	}
 }
 
-func TestRoundDownPowerOfTwo(t *testing.T) {
-	type testCase struct {
-		input int
-		want  int
+var sink any
```

Comment:
[question] should this be defined inside each Benchmark* test so that one benchmark does not impact another?